obspy/io/gcf/__init__.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
 """
-obspy.io.gcf - Guralp Compressed Format read support for ObsPy
-==============================================================
-This module provides read support for GCF waveform data and header info.
-Most methods are based on info from Guralp site
-http://geophysics.eas.gatech.edu/GTEQ/Scream4.4/GCF_Specification.htm
+obspy.io.gcf - Guralp Compressed Format, GCF, read and write support for ObsPy
+==============================================================================
+This module provides read and write support for `GCF<https://www.guralp.com/apps/ok?doc=GCF_Intro>` 
+waveform data and header info according to GCF Reference `SWA-RFC-GCFR Issue F, December 2021
+<https://www.guralp.com/apps/ok?doc=GCF_format>`.
```

Comment:
These links need to be reformatted, see https://docs.obspy.org/pr/gcf-c-source/packages/obspy.io.gcf.html#module-obspy.io.gcf

First link might need a space after "GCF", not sure. In any case, both links need to be postfixed with an underscore, see https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Reply:
Fixed (I think... Not sure how to build docs locally so will see after next push)